setup.cfg

```diff
@@ -1,5 +1,6 @@
 [pytest]
 pep8ignore = __init__.py E402
+-addopts = -v --basetemp=E:\mydir
```

Comment:
Yes it's only for windows..

On one of my dev-environment the python interpreter runs on E:\ but the tmp on win10 is on C:\ that runs not good.

The reasons therefore I could not find.
a) is the problem the relpath thing on win
b) User/Admin Permissions on win10 (env is on a workplace where i am not admin-rights!)

 Can pytest differentiate between win/linux for the cfg?